bluetooth/src/commonTest/kotlin/BluetoothUUIDTest.kt

```diff
@@ -65,10 +65,10 @@ class BluetoothUUIDTest: BaseTest() {
 
     @ExperimentalStdlibApi
```

Comment:
Not sure how it passed the first time with these annotations left in, but please remove them now.

Reply:
reminder they are still in, if you merge `0.2.0-rc` the setting is already so the annotations are not needed anymore.